test/e2e/profiles/recovery-mfa/.kratos.yml

```diff
@@ -0,0 +1,47 @@
+selfservice:
```

Comment:
For future record: unless you significantly need to alter a profile it's faster to use some default profile and change the config settings you need. Profiles can be slow to start up and introduce unnecessary duplication (like the identity schema)

Reply:
i see, i think we need to actually refactor how we do e2e tests in general. But i will keep this in mind :+1: